firebase/functions/src/index.ts

```diff
@@ -126,7 +152,7 @@ export const createUser = functions
             .auth()
             .setCustomUserClaims(userRecord.uid, {chainId: chainId});
       }
-      // TODO: Subscribe user in mailchimp if needed
+
       return {id: userRecord.uid};
```

Comment:
Shouldn't the user be subscribed here

Reply:
The issue referenced by this PR is indicating to subscribe the user when they join a loop, so not necessarily when they sign up (create user)